terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,34 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+api_local_url = "https://api.cidev.aws.chdev.org"
+authentication_middleware = "on"
+cache_server = "cidev-chs-elasticache.9vgugg.ng.0001.euw2.cache.amazonaws.com:6379"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
```

Comment:
And this one.

Reply:
the cdn_host seems to match the value we have in cidev profile for overseas-entities-web so should be ok